docs/cluster/cli.md

```diff
@@ -13,6 +13,7 @@ Flags:
   -e, --env string   environment (default "default")
   -f, --force        stop all running jobs
   -h, --help         help for deploy
+  -r, --refresh      re-deploy all APIs
```

Comment:
This comment is not consistent with full text for `cx deploy`. This comment is API focused whereas the full text just describes 'deployment state'. If we are changing our verbiage to be more API focused, we should change the full text as well.